src/services/comment.service.js

```diff
@@ -13,6 +13,28 @@ class CommentService {
   static createComment(newComment) {
     return Comments.create(newComment);
   }
+
+  /**
+   * @param {number} property
+   * @returns {find} this function finds comment
+  */
+  static findCommentByProperty(property) {
+    return Comments.findOne({
+      where: property
+    });
+  }
+
+  /**
+   * @param {object} property
+   * @param {object} data
+   * @returns {delete} this function deletes comment
+  */
+  static deleteCommentByProperty(property, data) {
+    return Comments.update(data, {
```

Comment:
There is a reason why you need to implement the soft delete feature properly. When done properly, you will be able to use Sequelize to only get rows that have not been deleted by default etc. 

Read here https://sequelize.org/master/manual/paranoid.html